dev-libs/roct-thunk-interface/roct-thunk-interface-9999.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
ditto

Reply:
Thanks for pointing out that EAPI=7 was possible now :)